src/api/commands/itemimage/ItemImageFeaturedCommand.ts

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) 2017-2019, The Particl Market developers
+// Distributed under the GPL software license, see the accompanying
+// file COPYING or https://github.com/particl/particl-market/blob/develop/LICENSE
+
+import { inject, named } from 'inversify';
+import { validate, request } from '../../../core/api/Validate';
+import { Logger as LoggerType } from '../../../core/Logger';
+import { Types, Core, Targets } from '../../../constants';
+import { ItemImageService } from '../../services/ItemImageService';
+import { RpcRequest } from '../../requests/RpcRequest';
+import { RpcCommandInterface } from '../RpcCommandInterface';
+import { MessageException } from '../../exceptions/MessageException';
+import { Commands} from '../CommandEnumType';
+import { BaseCommand } from '../BaseCommand';
+import { ListingItemTemplateService } from '../../services/ListingItemTemplateService';
+
+export class ItemImageFeaturedCommand extends BaseCommand implements RpcCommandInterface<void> {
```

Comment:
Rename this command to ListingItemTemplateFeatureImageCommand, since the first parameter is templateId and the commands purpose is to set the featured image for the template rather than just set the featured flag on an image.